pep-0631.rst

```diff
@@ -58,7 +58,9 @@ Every element SHOULD be an `entry <#entries>`_.
     ]
 
 Any element MAY be an inline table to support possible future functionality.
-Backends MUST silently ignore these until the specification is extended.
+Backends MUST error when encountering these until the specification is extended.
```

Comment:
Regarding the "Any element MAY be an inline table"... language, I fear this may be confusing to users who don't understand that the audience for this table is tools that are consumers of the `[project]` table, _not_ people writing the `[project]` table.

I'm not sure if there's better wording, though. Possibly:

> Future versions of this specification that allow for any element to be an inline table should be considered backwards compatible, and consumers of the `[project]` table should take this into consideration.